### ISSUE TITLE
Make "Unit Size" entry group changes through remaining catalogs

### ISSUE THREAD

**Issue by JimOrtiz** [BSData/warmahordes#40]
I like the feel of the changes @robbert229 & nberndt made in Khador for the units with min/max sizes. It lets the user see the unit size in a way that the "min unit -> max unit" upgrades don't.  I think we should pull this pattern into all the other catalogs.

**Comment by Obiken** [comment]
Yes, played a bit around with the Khador catalogue and really like hat way to show unit size and choice of UA.
